bullet_train-api/bullet_train-api.gemspec

```diff
@@ -38,7 +38,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "bullet_train"
 
   spec.add_dependency "rails", ">= 6.0.0"
-  spec.add_dependency "pagy", "~> 8"
+  spec.add_dependency "pagy", ">= 8.0"
```

Comment:
I'd be curious why we have the pessimistic operator here. Should we add a comment about the current reasoning for the version flip-flopping? 🩴